.gitignore

```diff
@@ -25,3 +25,6 @@ _testmain.go
 
 coverage.txt
 profile.out
+
+simplest-uncommitted-msg-0.1-jar-with-dependencies.jar
```

Comment:
where is this coming from? might be only for your environment right?

Reply:
nvm, not sure why github didn't show the entire diff, I see you're using it later.